src/filters/FiltersReducer.js

```diff
@@ -22,9 +22,6 @@ let initialState = {
 const filters = (state = initialState, action) => {
   switch (action.type) {
 
-  case types.DATA_CONFIG_REQUEST:
-    return {...state, loadingFilters: true};
```

Comment:
Using the spread operator without `Object.assign({}, {foo: bar})` seems to occur throughout this file - what's the reason to use the spread here?